packages/apollo-vscode/src/StatusBar.ts

```diff
@@ -0,0 +1,26 @@
+import { window, StatusBarItem, StatusBarAlignment } from "vscode";
+
+export default class ApolloStatusBar {
+  private statusBarItem: StatusBarItem = window.createStatusBarItem(
+    StatusBarAlignment.Right
+  );
+
+  constructor() {
+    this.statusBarItem.text = "Apollo GraphQL $(rss)";
+    this.statusBarItem.show();
+  }
+
+  public showLoadedState() {
+    if (!window.activeTextEditor) {
+      this.statusBarItem.hide();
+      return;
+    }
+
+    this.statusBarItem.text = "Apollo GraphQL $(rocket)";
```

Comment:
whoa, is this how you do emoji's with vscode?

Reply:
Yeah! These are the icons vscode makes available:
https://octicons.github.com/

Supports regular emoji too, just paste into code and all set 👍